demos/datepicker/icon-trigger.html

```diff
@@ -15,13 +15,17 @@
 				open: function() {
 					allowOpen = false;
 				}
-			});
+			} ),
+			widget = datepicker.datepicker( "widget" );
 
-		$( "<img src='images/calendar.gif' alt='Open Datepicker'>")
+		$( "<button>")
```

Comment:
Missing space before closing parent. I'm going to ignore other style issues, until we've got tooling to find these. 

@arschmitz what happened to your project to lint HTML embedded JS and CSS?